supervisor/const.py

```diff
@@ -441,3 +441,4 @@ class HostFeature(str, Enum):
     REBOOT = "reboot"
     SERVICES = "services"
     SHUTDOWN = "shutdown"
+    TIMEDATE = "timedate"
```

Comment:
In Python the package is called `datetime` so this name being the reverse is a bit confusing.

Reply:
Ah, see now it's the systemd name that's timedate.